tests/gflownet/envs/test_ccube.py

```diff
@@ -196,43 +196,43 @@ def test__mask_backward__1d__returns_expected(cube1d, state, mask_expected):
     [
         (
             [0.0, 0.0],
-            [True, False, True],
+            [True, False, True, False, False],
         ),
         (
             [0.5, 0.5],
-            [False, True, True],
+            [False, True, True, False, False],
         ),
         (
             [0.05, 0.5],
-            [True, False, True],
+            [True, False, True, False, False],
         ),
         (
             [0.5, 0.05],
-            [True, False, True],
+            [True, False, True, False, False],
         ),
         (
             [0.05, 0.05],
-            [True, False, True],
+            [True, False, True, False, False],
         ),
         (
             [0.90, 0.5],
-            [False, True, True],
+            [False, True, True, False, False],
         ),
         (
             [0.5, 0.90],
-            [False, True, True],
+            [False, True, True, False, False],
         ),
         (
             [0.95, 0.5],
-            [False, True, True],
+            [False, True, True, False, False],
         ),
         (
             [0.5, 0.95],
-            [False, True, True],
+            [False, True, True, False, False],
         ),
         (
             [0.95, 0.95],
-            [False, True, True],
+            [False, True, True, False, False],
```

Comment:
Should there be tests added with couple dimensions? Because, ATM, the tests all seem to expect the additional 'n_dim' dimensions to be False ?

Reply:
Ideally yes.